### DISCUSSION TITLE
Verifying Docker install

### DISCUSSION THREAD

**Discussion by meierthomas** [blakeblackshear/frigate#1057] [General]
I used the Docker install on a Pine64 running DietPI using the docker compose option. I created the file `docker-compose.yml` from the template provided and ran `docker-compose up -d` from the same directory the `docker-compose.yml` is in. Everything went as expected as far as I can tell.

Next step was to setup the custom component in Home Assistant. When trying to install the Frigate integration I specify the host as `http://<IP address of Pine64>:5000` and get an error `Unable to connect to Frigate`.

How can I verify the Docker installation and that Frigate is actually running on the Pine64?

On a related question if I need to make changes to the docker-compose file can I simply re-run the `docker-compose` command?

**Comment by blakeblackshear** [comment]
You should see the container listed when running `docker ps -a`. You can check the logs by running `docker logs <container_name>`. You should also be able to navigate to http://ip_of_pine64:5000 and view the UI.

**Reply by meierthomas** [reply]
Thanks for the response.

The container is in a restart loop due to some configuration issues. The log shows

```

Error parsing config: [Errno 2] No such file or directory: '/config/config.yml'
 * Starting nginx nginx
   ...done.
Error parsing config: [Errno 2] No such file or directory: '/config/config.yml'
 * Starting nginx nginx
   ...done.
```

and my docker-compose.yml file is setup as follows. Looks like I'm not understanding this syntax

```
 volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home:/config/config.yml:ro
      - /home/frigate:/media/frigate

```

`config.yml` is located at `/home/config`

Where do I go wrong?

**Reply by digiblur** [reply]
Looks like you are mapping a file to a folder.

**Reply by meierthomas** [reply]
Some updates.

I have `/config/config.yml`. I have the same file (i.c. config.yml) in `/home/config`. Then I renamed to `/config/config1.yml` and with the volume set to ` - /home/config/config.yml:/config/config.yml:ro` have rebuilt the container and restarted.

Now Frigate starts up on http://<IP address of Pine64>:5000.

What I'd like to understand though is why did this container start up after renaming the file to config1.yml. I know this is a straight Docker question but it would be interesting to understand nevertheless.

**Reply by blakeblackshear** [reply]
https://www.digitalocean.com/community/tutorials/how-to-share-data-between-the-docker-container-and-the-host

**Reply by meierthomas** [reply]
Thanks. A lot clearer. The system is up. Txs :)

**Reply by stahelp** [reply]
Hi

I struggle with the same error [Errno 21] Is a directory: '/config/config.yml when starting the container.

I have my config.yml in directory /config (contains just a section for mqtt and camera)
My docker-compose.yaml has these entries
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /config:/config/config.yml:ro
      - /media:/media/frigate
I also tried to use the CONFIG_FILE: /config variable, without success.

I do not see what I do wrong. Thank's in advance for any help.

**Reply by meierthomas** [reply]
This is what I ended up with for the container to start up

```
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home/config/config.yml:/config/config.yml:ro
      - /home/frigate:/home/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
```

**Comment by stahelp** [comment]
That helped me. Thank you

meierthomas ***@***.***> schrieb am Fr., 7. Mai 2021, 08:05:
